ipmininet/router/config/ospf.py

```diff
@@ -45,8 +44,9 @@ class OSPF(QuaggaDaemon):
     NAME = 'ospfd'
     DEPENDS = (Zebra,)
 
-    def __init__(self, *args, **kwargs):
-        super(OSPF, self).__init__(*args, **kwargs)
+    def __init__(self, node, routerid=None, *args, **kwargs):
+        super(OSPF, self).__init__(node=node, *args, **kwargs)
+        self.routerid = routerid
```

Comment:
What's that doing there? You don't want routers in an area to have the same router id ...